### ISSUE TITLE
Strain time threshold suggestion

### ISSUE THREAD

**Issue by PowerChaos125** [ppy/osu-performance#69]
Should we reconsider the 50ms cap in strain time calculation?
50ms corresponds to these limits: 600 bpm 1/2, 400 bpm 1/3, 300 bpm 1/4, 200 bpm 1/6
The limit I want to discuss is 300 bpm 1/4's, which is usually stream. 300 bpm map is rare. But with DT, such occurrence is considerable, as any >200 bpm map that contains 1/4's becomes >300 bpm map. Although such map is too difficult for most players, there is a (very) few can play. Regardless, since we allow many of such maps to be ranked, we should also ensure their playability too.  

I understand this cap is to prevent extreme cases. But I think we should raise the cap to include all ranked maps' bpm range including DT. As solution, setting this cap to 20ms (corresponds to 1500 bpm 1/2, 1000 bpm 1/3, 750 bpm 1/4, 500 bpm 1/6) is sufficient.

**Comment by Magnus-Cosmos** [comment]
I agree that 300bpm is becoming more and more possible these days, but raising it to 750bpm is way too much. One of the problem that arises from a larger cap is [this](https://www.youtube.com/watch?v=kDIX3Px0RiI), although stuff like that will probably never get ranked, it's should still be taken into consideration.

I'd say perhaps raising it to 40ms (375bpm) would be sufficient enough.

**Comment by VINXIS** [comment]
personally I think 600 bpm 1/4 should provide a significant boost to the speed diff, wouldn't be really nice to have 600 bpm be treated the same as half the bpm Lol 

20 ms is good because the game glitches out at any smaller ms interval due to some bug and it isn't really reasonable to limit higher bpms in the first place,
